src/som/interpreter/actors/EventualMessage.java

```diff
@@ -345,4 +366,24 @@ public boolean isBreakpoint() {
   public void setIsMessageReceiverBreakpoint(final boolean triggerBreakpoint) {
     this.triggerMessageReceiverBreakpoint = triggerBreakpoint;
   }
+
+  public boolean isStepOver() {
+    return this.triggerStepping == SteppingType.STEP_OVER;
+  }
+
+  public boolean isStepReturn() {
+     return this.triggerStepping == SteppingType.STEP_RETURN;
+  }
+
+  public boolean isStepInto() {
+    return this.triggerStepping == SteppingType.STEP_INTO;
+ }
+
+  public SteppingType getTriggerStepping() {
+    return this.triggerStepping;
+  }
```

Comment:
The `isStep*` methods and `getTriggerStepping()` seem to be somehow redundant.
Do we really need all variants. I'd think that we have either the `is*` methods, or `getTriggerStepping()`, but probably not both?